virttest/qemu_devices/qdevices.py

```diff
@@ -1390,7 +1390,11 @@ class Memory(QObject):
                                               "backend", "policy", "host-nodes",
                                               "share", "merge", "dump", "pmem",
                                               "discard-data", "align",
-                                              "prealloc-threads"]}
+                                              "prealloc-threads"],
+                      "memory-backend-memfd": ["size", "prealloc", "backend",
+                                               "seal", "policy", "host-nodes",
+                                               "share", "merge", "dump",
+                                               "hugetlb", "hugetlbsize"]}
```

Comment:
Just confirm, `prealloc-threads` and `x-use-canonical-path-for-ramblock-id` is not needed?

Reply:
I think so. They are not used by any cases now. I'd prefer to add if they are needed in the future.

Reply:
A good catch, i did not notice these 2 options since they are not in the qemu-kvm man page:
```
              -object                          memory-backend-memfd,id=id,merge=on|off,dump=on|off,share=on|off,preal‐
              loc=on|off,size=size,host-nodes=host-nodes,policy=default|preferred|bind|inter‐
              leave,seal=on|off,hugetlb=on|off,hugetlbsize=size
```
Sorry for that.